DFC.Integration.AVFeed/DFC.Integration.AVFeed.Service.AVAPI/AVAPIService.cs

```diff
@@ -83,12 +92,12 @@ public AVAPIService(IApprenticeshipVacancyApi apprenticeshipVacancyApi, IApplica
             }
 
             logger.Trace($"Extracting AV summaries for SOC {mapping.SocCode} page : {pageNumber}");
-           
+
             var queryString = HttpUtility.ParseQueryString(string.Empty);
 
-            queryString["standardLarsCodes"] = string.Join(",", mapping.Standards);
-            queryString["frameworkLarsCodes"] = string.Join(",", mapping.Frameworks);
-            queryString["pageSize"] = ConfigurationManager.AppSettings.Get("FAA.PageSize");;
+            queryString["standardLarsCodes"] = string.Join(",", mapping.Standards.Where(std=>!string.IsNullOrEmpty(std)));
```

Comment:
would be nice if you could following the styling guidelines and add spaces where needed to make it readable. :see_no_evil: